emsymbolizer.py

```diff
@@ -121,11 +153,8 @@ def decodeVLQ(string):
       if len(data) >= 4:
         col += data[3]
         info.append(col)
-      if len(data) >= 5:
```

Comment:
Maybe leave a `# TODO: see if we need the name, which is the next field` - ?

Reply:
Done